templates/gax/ruby/README.md.mustache

```diff
@@ -0,0 +1,36 @@
+GAX library for the Google {{api.simplename}} API
+=================================================
+
+{{api.name}}-{{api.version}} uses [google-gax][Google API extensions] to provide an
+easy-to-use client library for the [`Google {{api.simplename}} API`][] ({{api.version}}) defined in the [googleapis][] git repository
```

Comment:
FYI: I recommend s/api.simplename/api.titlename/ for use in text now that #26 is checked in.

The main difference is that titlename is capitalized, the resulting text reads a bit better.